conbench/app/benchmarks.py

```diff
@@ -95,6 +95,11 @@ def _get_info(self, benchmark):
 class RunMixin:
     def get_display_run(self, run_id):
         run, response = self._get_run(run_id)
+
+        if response.status_code == 404:
+            self.flash(f"Run ID unknown: {run_id}", "info")
+            return None
```

Comment:
Table stakes. Need to clean up things like that along the way for our own sanity.